modules/tracking/src/trackerKCF.cpp

```diff
@@ -87,18 +87,20 @@ namespace cv{
     * KCF functions and vars
     */
     void createHanningWindow(OutputArray _dst, const cv::Size winSize, const int type) const;
-    void inline fft2(const Mat src, std::vector<Mat> & dest) const;
-    void inline fft2(const Mat src, Mat & dest) const;
+    void inline fft2(const Mat src, std::vector<Mat> & _dest, std::vector<Mat> & _layers) const;
```

Comment:
i forget to clean up this mess, because i redefined several variables with duplicated name (from inside function become public variable), i will clean this mess soon